curriculum/challenges/english/blocks/lecture-working-with-arrays-variables-and-naming-practices/6732c6ba2ea42b610b9f9ce1.md

```diff
@@ -122,41 +102,41 @@ Remember that the `length` property returns the highest numeric index plus one,
 What will be the output of the following code?
```

Comment:
even if we are removing the interactive editor from here, maybe the question should not use the same difference between a sparse array and an array with `undefined` values that is difficult to see and looks confusing depending on the environment

Reply:
it does not look like this behaviour of `Array.from` is taught in this lesson